README.md

```diff
@@ -52,18 +52,18 @@ Each specification is described by a JSON object that will allow retrieving info
-* in case the reference to the specification would benefit from being more specific than the specification as a whole, the `feature` property allows to add the name of the specific feature (see e.g. the [reference to the HTMLMediaElement interface in the HTML5 specification](data/htmlmediaelement.json))
-* for specifications that have not started their Recommendation track progress, the `title` property gives the title of the specification
-* for specifications that have not started their Recommendation track progress, the `editors` property should point to the URL of the editors draft of the specification
-* for specifications that have not started their Recommendation track progress, the `wgs` property should be an array of objects describing the groups that are producing the spec; each such object should have a `url` property with a link to the group's home page, and a `label` property with the name of the group
+* `url`: should point to the URL of the latest version of the specification. This URL will be used to collect additional data about the spec (standardization status, Working Groups that produce it, editors draft, etc) and as target of links that reference the feature. URL may contain a fragment to point to a specific section in a specification. If the `url` property is not specified, the framework assumes that the underlying specification is a W3C specification,  that the filename is its short name, and that the URL of the spec is `https://www.w3.org/TR/[filename]/`.
+* `impl`: for specifications for which browser implementations are expected, the `impl` property explains where to look for implementation info. Described below in [Describing implementation status](#describing-implementation-status)
+* `polyfills`: for specifications for which there are polyfills available that would be worth reporting, the `polyfills` property lists these polyfills. It should be an array of objects that have a `url` property that links to the polyfill's home page on the Web, and a `label` property with the name of polyfill.
+* `feature`: in case the reference to the specification would benefit from being more specific than the specification as a whole, the `feature` property allows to add the name of the specific feature (see e.g. the [reference to the HTMLMediaElement interface in the HTML5 specification](data/htmlmediaelement.json))
+* `title`: when the specification is unknown to the [W3C API](https://w3c.github.io/w3c-api/) and to [Specref](https://www.specref.org/), the `title` property should be set to the title of the specification
+* `edDraft`: when the specification is unknown to the [W3C API](https://w3c.github.io/w3c-api/) and to [Specref](https://www.specref.org/), or when these APIs do not know the URL of the Editor's Draft for the specification, the `edDraft` property should contain the URL of the Editor's Draft of the specification.
```

Comment:
Why not use `editors` like before?

Reply:
Because I'd like to be able to use `editors` later on to list the editors of a specification, as done in ReSpec (actually, I hesitated to name the property `edDraftURI` as in ReSpec, but `URI` seemed a bit pompous).

Reply:
`edDraft` sounds like... `editors' draft Draft` to me. It depends on how you interpret the lowercase letter "d". But anyway, it doesn't matter that much and I think I can live with this ^_^

Reply:
Would `editorDraftUrl` or `editorDraft` look better for you? I don't really like `edDraft` either, I just wanted to remain close to what Respec uses. But then, since I already dropped `URI`...

Reply:
`editorsDraft` looks better to me, but maybe we can apply the principle of least astonishment for ReSpec users here. (I didn't think of the ReSpec issue.)